front50-s3/src/main/java/com/netflix/spinnaker/front50/config/S3Config.java

```diff
@@ -71,10 +72,15 @@ public AmazonS3 awsS3Client(AWSCredentialsProvider awsCredentialsProvider, S3Pro
     }
 
     AmazonS3Client client = new AmazonS3Client(awsCredentialsProvider, clientConfiguration);
-    Optional.ofNullable(s3Properties.getRegion())
-      .map(Regions::fromName)
-      .map(Region::getRegion)
-      .ifPresent(client::setRegion);
+    if (s3Properties.getEndpoint() != null){
```

Comment:
Minor code formatting ... (spacing between brackets)

```
if (s3Properties.getEndpoint != null) {
...
} else { 
...
}
```

If you could fix that, I'll get this merged in.